### PR TITLE
[zh] Updated some text in setup-kubernetes-cluster

### PR DESCRIPTION
**Pull request by Catherine-monk**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by wilsonwu** [comment]
Seems not very necessary for these updates, @windsonsea PTAL.

File: content/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ test: no
     {{< /text >}}
 
     {{< tip >}}
-    如果您是一位讲师，可以为每个参与者分配独立的命名空间。本教程支持多个参与者在不同的命名空间下同时运行。
+    如果您是管理员，您可以为每个用户分配一个独立的命名空间。本教程支持多个用户同时在多个命名空间中进行工作。
```
Comment (windsonsea) [inline content/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md:35]: You may need to change the en first if you believe the prospective of `admin` is better.

File: content/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ test: no
     - [Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/)
     - [Kiali](https://www.kiali.io/documentation/getting-started/)
 
-    `kubectl` 命令可以接收一个行内配置去为每个服务创建 Ingress 资源：
+    `kubectl` 命令可以通过修改 yaml 配置为每个服务创建 Ingress 资源：
```
Comment (windsonsea) [inline content/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md:58]: Same issue. This change will cause a gap to en original.

**Comment by windsonsea** [comment]
/retitle [zh] Updated some text in setup-kubernetes-cluster

**Comment by windsonsea** [comment]
/re-title [zh] Updated some text in setup-kubernetes-cluster

**Comment by windsonsea** [comment]
ping @Catherine-monk

**Comment by windsonsea** [comment]
You need to sign CLA before the review process.

**Review by windsonsea — APPROVED** [review]
Lets kick this in. The [preview](https://deploy-preview-13604--preliminary-istio.netlify.app/latest/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/) looks not bad.

Hi @Catherine-monk, thank you for your contribution.

I have a quick tip for you: if you're unfamiliar with the workflow, consider making minor changes instead of major ones that may cause issues such as lint failed or genchecks as below.

File: content/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md [hunk 3]
```diff
@@ -34,8 +34,12 @@ test: no
     {{< /text >}}
 
     {{< tip >}}
+<<<<<<< HEAD
```
Comment (hanxiaop) [inline content/zh/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md:37]: You need to resolve these conflicts to pass lint

**Pull request merged**